pkg/agent/openflow/client.go

```diff
@@ -465,19 +466,11 @@ func (c *client) UninstallServiceFlows(svcIP net.IP, svcPort uint16, protocol bi
 }
 
 func (c *client) InstallLoadBalancerServiceFromOutsideFlows(svcIP net.IP, svcPort uint16, protocol binding.Protocol) error {
-	c.replayMutex.RLock()
-	defer c.replayMutex.RUnlock()
-	var flows []binding.Flow
-	flows = append(flows, c.loadBalancerServiceFromOutsideFlow(svcIP, svcPort, protocol))
-	cacheKey := fmt.Sprintf("LoadBalancerService_%s_%d_%s", svcIP, svcPort, protocol)
-	return c.addFlows(c.serviceFlowCache, cacheKey, flows)
+	return c.installLoadBalancerServiceFromOutsideFlows(svcIP, svcPort, protocol)
```

Comment:
We do not have client_linux.go but only pipeline_os.go, so feel better to keep funcs defined in Client interface in client.go. Do you prefer to add client_os.go? I hope to avoid adding more _os.go files.

Reply:
InstallLoadBalancerServiceFromOutsideFlows doesn't sound like a part of pipeline. I would prefer adding client_os.go.

Reply:
Added client_os.go.